src/qt/forms/communityfundcreateproposaldialog.ui

```diff
@@ -0,0 +1,246 @@
+       </property>
+       <property name="text">
+        <string>Proposal Duration:</string>
+       </property>
+      </widget>
+     </item>
+     <item>
+      <layout class="QHBoxLayout" name="horizontalLayoutDuration">
+       <item>
+        <layout class="QVBoxLayout" name="verticalLayoutDays">
+         <item>
+          <widget class="QValidatedSpinBox" name="spinBoxDays">
+           <property name="value">
+            <number>0</number>
+           </property>
+          </widget>
+         </item>
+         <item>
+          <widget class="QLabel" name="labelDaysTitle">
+           <property name="text">
+            <string>Days</string>
+           </property>
+           <property name="alignment">
+            <set>Qt::AlignCenter</set>
+           </property>
+          </widget>
+         </item>
+        </layout>
+       </item>
+       <item>
+        <layout class="QVBoxLayout" name="verticalLayoutHours">
+         <property name="leftMargin">
+          <number>0</number>
+         </property>
+         <item>
+          <widget class="QValidatedSpinBox" name="spinBoxHours"/>
+         </item>
+         <item>
+          <widget class="QLabel" name="labelHoursTitle">
+           <property name="text">
+            <string>Hours</string>
+           </property>
+           <property name="alignment">
+            <set>Qt::AlignCenter</set>
+           </property>
+          </widget>
+         </item>
+        </layout>
+       </item>
+       <item>
+        <layout class="QVBoxLayout" name="verticalLayoutMinutes">
+         <property name="leftMargin">
+          <number>0</number>
+         </property>
+         <item>
+          <widget class="QValidatedSpinBox" name="spinBoxMinutes"/>
+         </item>
+         <item>
+          <widget class="QLabel" name="labelMinutesTitle">
+           <property name="text">
+            <string>Minutes</string>
+           </property>
+           <property name="alignment">
+            <set>Qt::AlignCenter</set>
+           </property>
+          </widget>
+         </item>
+        </layout>
+       </item>
+      </layout>
+     </item>
+     <item>
+      <layout class="QHBoxLayout" name="horizontalLayoutCreateProposalButton">
+       <item>
+        <widget class="QPushButton" name="pushButtonCreateProposal">
+         <property name="text">
+          <string>Create Proposal</string>
+         </property>
+        </widget>
+       </item>
+       <item>
+        <widget class="QPushButton" name="pushButtonClose">
+         <property name="text">
+          <string>Close</string>
+         </property>
+        </widget>
+       </item>
+       <item>
+        <widget class="QLabel" name="labelWarning">
+         <property name="font">
+          <font>
+           <pointsize>8</pointsize>
+          </font>
+         </property>
+         <property name="text">
+          <string>By submitting the proposal a 50 NAV deduction will occur from your wallet </string>
```

Comment:
ideally those consensus-related values should be automatically taken from Params().GetConsensus()